pom.xml

```diff
@@ -223,6 +223,11 @@
         <version>1.1.1</version>
       </dependency>
       <dependency>
+        <groupId>commons-pool</groupId>
+        <artifactId>commons-pool</artifactId>
+        <version>1.6</version>
```

Comment:
Should we be trying to use org.apache.commons:commons-pool2 instead? I'm not sure if commons-pool is still receiving updates.